src/python/CMakeLists.txt

```diff
@@ -37,14 +37,10 @@ foreach(PY_CMD ${PYTHON_CMD})
   )
   add_custom_target(bcc_py_${PY_CMD_ESCAPED} ALL DEPENDS ${PIP_INSTALLABLE})
 
-  if(NOT PYTHON_PREFIX)
-     set(PYTHON_PREFIX, ${CMAKE_INSTALL_PREFIX} )
-  endif()
```

Comment:
Can you specify the PYTHON_PREFIX explicitly in order to install at your desired place?
The following patch
  https://github.com/iovisor/bcc/commit/6813fbce8718bede5075272cdc3ab9314eb91b5c
introduced the feature.